_config_dev.yml

```diff
@@ -4,5 +4,7 @@ plugins:
 markdown: redcarpet
 highlighter: rouge
 
-baseurl: ''
-clsbaseurl: /cls
\ No newline at end of file
+url: https://snoopycrimecrop.github.io
```

Comment:
Why are these changes necessary?

Reply:
Not totally sure if this helps, but the link rings a bell with https://github.com/ome/ome-help/pull/332#issuecomment-652985463 - this is possibly a way which @jburel put in place to be able to check the redirects ? The redirects work if ``https://snoopycrimecop.github.io/ome-help``  is added to the url as a prefix. 

Also, if I build this repo locally from  master branch (not jbure/master,  origin/master), then I can prefix the links with http://0.0.0.0:4000/ and it works too.

But if I try to build from jburel/master, then the docker command fails.

Reply:
the change is also in origin/master
This is what I commit when investigating the staging work.
This might no longer be required

Reply:
OK, now I am seeing https://github.com/ome/ome-help/pull/332 which was merged 6 days ago. Somehow this comes back to the question I asked in https://github.com/ome/ome-help/pull/334#pullrequestreview-464322757 i.e. as far as I can tell this work has already been reviewed and merged into `master` but the issue is we cannot merge into `gh-pages` right? Should we try to fix the conflict? Or simply ditch the current workflow?

Reply:
to bring it to prod in the current workflow, we have to merge it to gh-pages.
Let's discuss tomorrow